packages/docusaurus-plugin-content-docs/src/props.ts

```diff
@@ -70,9 +70,25 @@ Available document ids are:
     }
   }
 
-  function convertCategory(item: SidebarItemCategory): PropSidebarItemCategory {
+  function convertCategory(
+    item: SidebarItemCategory,
+  ): PropSidebarItemCategory | PropSidebarItemLink {
     const {link, ...rest} = item;
     const href = getCategoryLinkHref(link);
+    // If the current category doesn't have subitems, we render a normal doc link instead.
+    if (item.items.length === 0) {
```

Comment:
It works but I don't think it's the best place to add this code.

We'd rather move this to the generator logic itself, so that sidebar data is clean earlier in the docs plugin lifecycle.

Later when users will try to extend the docs plugin, they'll be happy to receive proper sidebar data in the `content` param, which is not the case with current impl